MIGRATION.md

```diff
@@ -0,0 +1,21 @@
+## Migration
+
+### Migrating from v3 to v4
+
+V4 has dropped support for use through Bower. When using Sass, you will need to add `node_modules` to your `includePath` as well as either:
+- Update your code to use the fully qualified package name, i.e.: `@import "@financial-times/o-brand/main"`, or
+- Also `node_modules/@financial-times` to the Sass include path
+
+### Migrating from v2 to v3
```

Comment:
Were these guides missing? Nice one adding them 👍

Reply:
i think you did

Reply:
oh